server/datastore/mysql/software.go

```diff
@@ -119,23 +119,58 @@ func nothingChanged(current, incoming []fleet.Software, minLastOpenedAtDiff time
 	return true
 }
 
+func (ds *Datastore) ListSoftwareByHostIDShort(ctx context.Context, hostID uint) ([]fleet.Software, error) {
```

Comment:
Missing smoke tests for `Datastore.ListSoftwareByHostIDShort` in `mysql` package (in `software_test.go`).

Reply:
Added in 9152bad6c